lib/data_pull.rb

```diff
@@ -20,6 +21,7 @@ def initialize(argv:, stdout:, stderr:)
     :include_missing,
     :format,
     :show_help,
+    :requesting_issuers, # Array of issuers
```

Comment:
Sorry to be coming in with petty nits, but -- I'm not sure this is a helpful comment. Without reading further below I had no idea what a `requesting_issuer` was, and "Array of issuers" didn't help.

Reply:
So would it be clearer to remove the comment?

Reply:
will remove
```suggestion
    :requesting_issuers,
```